mappings/net/minecraft/client/item/ModelPredicateProviderRegistry.mapping

```diff
@@ -0,0 +1,45 @@
+CLASS net/minecraft/class_5272 net/minecraft/client/item/ModelPredicateProviderRegistry
+	FIELD field_24443 GLOBAL Ljava/util/Map;
+	FIELD field_24444 DAMAGED_ID Lnet/minecraft/class_2960;
+	FIELD field_24445 DAMAGE_ID Lnet/minecraft/class_2960;
+	FIELD field_24446 DAMAGED_PROVIDER Lnet/minecraft/class_1800;
+	FIELD field_24447 DAMAGE_PROVIDER Lnet/minecraft/class_1800;
+	FIELD field_24448 ITEM_SPECIFIC Ljava/util/Map;
+	METHOD method_27878 get (Lnet/minecraft/class_1792;Lnet/minecraft/class_2960;)Lnet/minecraft/class_1800;
+		ARG 0 item
+		ARG 1 id
+	METHOD method_27879 register (Lnet/minecraft/class_1792;Lnet/minecraft/class_2960;Lnet/minecraft/class_1800;)V
+		ARG 0 item
+		ARG 1 id
+		ARG 2 provider
+	METHOD method_27881 register (Lnet/minecraft/class_2960;Lnet/minecraft/class_1800;)Lnet/minecraft/class_1800;
+		ARG 0 id
+		ARG 1 provider
+	CLASS 1
+		FIELD field_7910 step D
+		FIELD field_7911 time D
+		FIELD field_7913 lastTick J
+		METHOD method_7736 getTime (Lnet/minecraft/class_1937;D)D
+			ARG 1 world
+			ARG 2 skyAngle
+	CLASS 2
```

Comment:
You don't name these classes?

Reply:
They're anonymous classes if their name is just a number.